airbyte-integrations/connectors/source-dynamodb/acceptance-test-config.yml

```diff
@@ -0,0 +1,30 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-dynamodb:dev
+tests:
+  spec:
+    - spec_path: "source_dynamodb/spec.yaml"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      empty_streams: []
+  # TODO uncomment this block to specify that the tests should assert the connector outputs the records provided in the input file a file
+  #      expect_records:
+  #        path: "integration_tests/expected_records.txt"
+  #        extra_fields: no
+  #        exact_order: no
+  #        extra_records: yes
+  # incremental: # TODO if your connector does not implement incremental sync, remove this block
+  #   - config_path: "secrets/config.json"
+  #     configured_catalog_path: "integration_tests/configured_catalog.json"
+  #     future_state_path: "integration_tests/abnormal_state.json"
+  # full_refresh:
+  #   - config_path: "secrets/config.json"
+  #     configured_catalog_path: "integration_tests/configured_catalog.json"
```

Comment:
You need to re-enable this and run integration test again.

Reply:
work in progress...